Makefile

```diff
@@ -2,14 +2,18 @@
 # with Go source code. If you know what GOPATH is then you probably
 # don't need to bother with make.
 
-.PHONY: geth mist all test travis-test-with-coverage clean
+.PHONY: geth evm mist all test travis-test-with-coverage clean
 GOBIN = build/bin
 
 geth:
-	build/env.sh go install -v $(shell build/ldflags.sh) ./cmd/geth
+	build/env.sh $(GOROOT)/bin/go install -v $(shell build/ldflags.sh) ./cmd/geth
```

Comment:
I am guessing that you added this in order to be able to build/test with different Go versions.
If that's what you want, we can add a variable `GO` that allows you to specify the go binary.
You could then invoke make as follows:

```
make GO=/path/to/go/bin/go1.5
// or even
make GO=go1.5
```